root/static/scripts/edit/MB/CoverArt.js

```diff
@@ -204,12 +204,12 @@ MB.CoverArt.file_data_uri = function (file) {
     return deferred.promise();
 };
 
-MB.CoverArt.sign_upload = function (file, gid, mime_type) {
+MB.CoverArt.sign_upload = function (file, gid, mimeType) {
     var deferred = $.Deferred();
 
     var postfields = $.ajax({
         url: "/ws/js/cover-art-upload/" + gid,
-        data: { mime_type: mime_type },
+        data: { mimeType: mimeType },
```

Comment:
The data being sent to the server can't be changed here, 'cause the server won't understand it :)
```suggestion
        data: { mime_type: mimeType },
```